pkg/webhookadmission/utils.go

```diff
@@ -205,15 +226,23 @@ func createWebhookV1(caBundle []byte, ns string) error {
 		Webhooks: []admissionv1.MutatingWebhook{webhook},
 	}
 
-	// recreate webhook
-	err := admissionregistration.Instance().DeleteMutatingWebhookConfiguration(storkAdmissionController)
-	if err != nil && !k8serr.IsNotFound(err) {
+	// UpdateOrCreate v1 webhook
+	resp, err := admissionregistration.Instance().GetMutatingWebhookConfiguration(storkAdmissionController)
+	if err != nil {
+		if k8serr.IsNotFound(err) {
+			_, err = admissionregistration.Instance().CreateMutatingWebhookConfiguration(req)
+			if err != nil {
+				log.Errorf("Unable to create webhook configuration: %v", err)
+			}
+		}
 		return err
```

Comment:
Do we want to return an error for the `IsNotFound` case?

Reply:
If `IsNotFound`, our goal is to recreate it. We don't want to return error in that case.

Reply:
But, in this case if there `IsNotFound()` condition is followed, then the `CreateMutatingWebhookConfiguration()` function will get called and post that the control will go to line 238 where it shall return the error.

Reply:
err will get updated with the returned err val from CreateMutatingWebhookConfiguration(). If it's nil, nil will be returned.

Reply:
Oh yes. My bad.